epa-under-siege-peer-reviews.html

```diff
@@ -43,7 +54,7 @@ <h2 class="f3 fw3 ttu white-80">The EPA Under Siege: Trump’s Assault in Histor
             <div id="contr" class="vh-25 w-100 dt tc tile pv0-l pv2-m pv4-ns">
                 <div class="tile-content">
                     <h3 class="f3 ttu white">Contributors</h3>
-                    <p class="f4 fw2 white lh-normal">Christopher Sellers, Lindsey Dillon, Jennifer Liss Ohayon, Nick Shapiro, Marianne Sullivan, Chris Amoss, Stephen Bocking, Phil Brown, Vanessa De la Rosa, Jill Harrison, Sara Johns, Katherine Kulik, Rebecca Lave, Michelle Murphy, Liza Piper, Lauren Richter, Sara Wylie</p>
+                    <p class="f4 fw2 white lh-normal">Dr. Richard Andrews, Dr. Nancy Langston, Dr. Jay Turner</p>
```

Comment:
Can anyone tell me if this reduction in names was intentional? It's happening in a commit whose message doesn't reflect this being the intention